packages/vite/src/node/plugins/html.ts

```diff
@@ -606,13 +608,16 @@ export function buildHtmlPlugin(config: ResolvedConfig): Plugin {
       for (const [id, html] of processedHtml) {
         const relativeUrlPath = path.posix.relative(config.root, id)
         const assetsBase = getBaseInHTML(relativeUrlPath, config)
-        const toOutputAssetFilePath = (filename: string) => {
+        const toOutputAssetFilePath = (
+          filename: string,
+          type: 'asset' | 'public' = 'asset'
+        ) => {
```

Comment:
Maybe it is better to rename this function to `toOutputFilePath` and remove default value from `type` parameter and add a `toOutputAssetFilePath` function below? (because this function now handles non-asset files)
```js
const toOutputAssetFilePath = (
  filename: string
) => toOutputAssetFilePath(filename, 'asset')
```

Reply:
@sapphi-red Do you mean like this?

```diff
const toOutputAssetFilePath = (
  filename: string
-) => toOutputAssetFilePath(filename, 'asset')
+) => toOutputFilePath(filename, 'asset')
```

If so, I pushed the change. Please help to check it again. Thanks.

Reply:
My bad. Yes, I meant that. Thank you!